scripts/linters/js_ts_linter.py

```diff
@@ -74,7 +86,8 @@ def _parse_js_or_ts_file(filepath, file_content, **kwargs):
 
 
 def _get_expression_from_node_if_one_exists(
-        parsed_node, components_to_check):
+    parsed_node: esprima.nodes.Node, components_to_check: List[str]
+) -> esprima.nodes.Node:
     """This function first checks whether the parsed node represents
     the required angular component that needs to be derived by checking if
     its in the 'components_to_check' list. If yes, then it  will return the
```

Comment:
its --> it's

Drop extra space before "will"

Reply:
Done!